src/components/AssociatedKeywords.js

```diff
@@ -0,0 +1,22 @@
+import React from "react";
+import * as s from "./SearchKeyWords.cs";
+import SingleSearchKeyword from "./SingleSearchKeyword";
+
+export default function AssociatedKeywords({ associatedKeywords }) {
```

Comment:
This is never used because we don't have a backend for it yet... I really wonder whether we can remove it for now. You might want to save it on your machine somewhere, in case we move on to implement it, but until then it's going to be a little bit of dead code: i'm not sure it's useful to have it. What do you think?

Reply:
I have saved the implementation locally, and have removed all of the files from the PR.